testing/integration/monitoring_logs_test.go

```diff
@@ -92,15 +79,15 @@ func (runner *EnrollRunner) TestEnroll() {
 		require.NoError(t, err)
 	})
 
-	t.Logf("sleeping for one minute...")
-	time.Sleep(time.Second * 60)
+	AssertConnectedToFleet(t, agentFixture)
```

Comment:
Now that this helper function is being used by multiple tests, could we move it's definition out of `proxy_url_test.go` and into somewhere under https://github.com/elastic/elastic-agent/tree/main/pkg/testing/tools?